test/OWNERS

```diff
@@ -1,10 +1,6 @@
 approvers:
-  - vicaire
-  - IronPan
```

Comment:
why removing me? :)

Reply:
You are already the owner on the top level, which naturally means you are the approver for everything. If you want to keep your name here, I don't have problems with that though.

Reply:
this would help people triage the issue at finer level. could you add it back?

Reply:
Done.

Reply:
can u add me back to the approver?

Reply:
?? You are still in the approver list. 
Can you check again? Thanks

Reply:
nvm order changed i crossed my eyes.